concise-mid-tag.cddl

```diff
@@ -160,16 +138,22 @@ module-version-map = {
 
 version-type = text .default '0.0.0'
 
-; Notes:
-; - do we need an extension point in element-value given
-;   there are extension points in endorsed-value and
-;   reference-value?
 element-value-group = (
   ? comid.version => module-version-map
   ? comid.svn => svn-type
   ? comid.digests => digests-type
   ? comid.flags => flags-type
   ? raw-value-group
+;
+; former $$instance-value-group-choice, except it's not
```

Comment:
It seems wrong to include instance values like Mac-addr in element-value-group because these are always instance data. Element-value-group is most often class data, but could be both. The data structure should not encourage mixing instance data with class data IMO.